Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/disassembler/DisassemblerPlugin.java

```diff
@@ -301,19 +301,6 @@ boolean checkDisassemblyEnabled(ListingActionContext context, Address address, b
 		return currentProgram.getMemory().contains(address);
 	}
 
-	/**
-	 * @see ghidra.app.plugin.contrib.disassembler.DisassemblyTaskListener#disassembleMessageReported(String)
-	 */
-	public void disassembleMessageReported(String msg) {
-		tool.setStatusInfo(msg);
-	}
-
-	/**
-	 * @see ghidra.app.plugin.contrib.disassembler.DisassemblyTaskListener#disassemblyDone(DisassemblyTask)
-	 */
-	public void disassemblyDone(Disassembler task) {
-	}
-
```

Comment:
For the top one the interface would now be DisassemblerMessageListener. A dev needs to chime in for whether or not these are actually safe to be removed. They might be accessed via reflection.

Reply:
Hm I removed these out of thé following thinking: Seems like they are not directly used and also reference something with `@see` which doesn't exist any longer. I wanted to fix the javadoc only first, but then I though that they should have been overriding something in a previous version, which has been removed. We can revert this, but the javadoc needs to be removed for sure.